src/widgets/spinbox.cpp

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// SPDX-FileCopyrightText: 2017-2019 Alejandro Sirgo Rica & Contributors
+
+#include "src/widgets/spinbox.h"
+#include "src/utils/confighandler.h"
+
+SpinBox::SpinBox(QWidget* parent)
```

Comment:
Im not sure SpinBox is the best name for this. Maybe put something in the name about what makes it different from a normal spin box? Like ColorSpinBox or something